### DISCUSSION TITLE
*忽略 Java 兼容性警告*选项对这段代码管用吗：

### DISCUSSION THREAD

**Discussion by zkitefly** [Meloong-Git/PCL#2229] [提问] [answered]
*忽略 Java 兼容性警告*选项对这段代码管用吗：

https://github.com/Hex-Dragon/PCL2/blob/2c2bd2d4893417d9b10e270c9cc82aceea747bbf/Plain%20Craft%20Launcher%202/Modules/Minecraft/ModJava.vb#L132-L133

![image](https://github.com/Hex-Dragon/PCL2/assets/64117916/c5eb747d-c1f4-4ff5-a6c4-3172eaa63be3)

**Comment by Pigeon0v0** [comment]
我觉得不如让人换成 JDK 得了…

**Reply by hejiehao** [reply]
那这个只防JRE不防JDK有什么用呢？
安装JDK但是玩家又没有开发需求那不就会浪费玩家的硬盘空间

**Reply by Pigeon0v0** [reply]
> 那这个只防JRE不防JDK有什么用呢？
> 安装JDK但是玩家又没有开发需求那不就会浪费玩家的硬盘空间

这个是 JRE 16 独有的特性，JDK 16 没有这个问题

而且现在谁还放不起一个 JDK ..

**Reply by hejiehao** [reply]
> > 那这个只防JRE不防JDK有什么用呢？
> > 安装JDK但是玩家又没有开发需求那不就会浪费玩家的硬盘空间
> 
> 这个是 JRE 16 独有的特性，JDK 16 没有这个问题
> 
> 而且现在谁还放不起一个 JDK ..

所有发行版都这样吗？

**Reply by safesaffawa** [reply]
但是 硬盘鬼不认为 他们把自己的硬盘如视珍宝 这些多一点可是要他们的魂命

**Comment by LTCatt** [comment, marked as answer]
换用 JDK 即可
